### PR TITLE
add takeOnce, add last update date in meta

### PR DESCRIPTION
**Pull request by Panacee**

### CODE REVIEW

File: src/createMetaReducer.js [hunk 1]
```diff
@@ -2,15 +2,17 @@ import { toSuccess, toReset, toError, toRequest } from './actionHelpers';
...
 
 export default function createMetaReducer(actionName) {
     return (state = { [defaultKey]: defaultState }, { type, payload, params, requestAction }) => {
         const param = pathOr(params || noParamsKey, ['params'], requestAction);
 
         const getState = (pending, error, success) => {
-            const meta = { pending, error, success };
-            return assoc(getPath(param), meta, state);
+            const path = getPath(param);
+            const lastUpdate = success ? new Date().toString() : pathOr(null, [path, 'lastUpdate'], state);
```
Comment (megazoll) [inline src/createMetaReducer.js]: `(new Date).toISOString()`

Comment (Panacee) [reply src/createMetaReducer.js]: done

File: src/createMetaReducer.js [hunk 2]
```diff
@@ -2,15 +2,20 @@ import { toSuccess, toReset, toError, toRequest } from './actionHelpers';
...
 export default function createMetaReducer(actionName) {
     return (state = { [defaultKey]: defaultState }, { type, payload, params, requestAction }) => {
         const param = pathOr(params || noParamsKey, ['params'], requestAction);
 
         const getState = (pending, error, success) => {
-            const meta = { pending, error, success };
-            return assoc(getPath(param), meta, state);
+            const path = getPath(param);
+            const reset = !pending && !error && !success;
+            const lastSucceedAt = reset ? null :
```
Comment (megazoll) [inline src/createMetaReducer.js]:
```js
export default function createMetaReducer(actionName) {
    return (state = { [defaultKey]: defaultState }, { type, payload, params, requestAction }) => {
        const param = pathOr(params || noParamsKey, ['params'], requestAction);
        const getLastSucceedAt = () => pathOr(null, [getPath(param), 'lastSucceedAt'], state);

        const getState = (pending, error, success, lastSucceedAt = getLastSucceedAt()) => {
            const meta = { pending, error, success, lastSucceedAt };
            return assoc(path, meta, state);
        };

        switch(type) {
            case toRequest(actionName):
                return getState(true, false, false);
            case toSuccess(actionName):
                return getState(false, false, true, (new Date).toISOString());
            case toError(actionName):
                return getState(false, payload, false);
            case toReset(actionName):
                return getState(false, false, false, null);
            default:
                return state;
        }
    };
}
```

Comment (Panacee) [reply src/createMetaReducer.js]: +

**Pull request merged**